<testsuites id="" name="" tests="1" failures="1" skipped="0" errors="0" time="70.34150900000007">
<testsuite name="e2e\permissons\add-permissons.spec.ts" timestamp="2025-03-25T03:46:12.805Z" hostname="chromium" tests="1" failures="1" skipped="0" time="60.994" errors="0">
<testcase name="[US05] Asignación de permisos | Asignar permisos a los empleados › Validar que no se puede crear un nuevo permiso con un nombre de usuario ya existente" classname="e2e\permissons\add-permissons.spec.ts" time="60.994">
<failure message="add-permissons.spec.ts:68:9 Validar que no se puede crear un nuevo permiso con un nombre de usuario ya existente" type="FAILURE">
<![CDATA[  [chromium] › e2e\permissons\add-permissons.spec.ts:68:9 › [US05] Asignación de permisos | Asignar permisos a los empleados › Validar que no se puede crear un nuevo permiso con un nombre de usuario ya existente › Deberia aparecer un mensaje de error 

    Test timeout of 60000ms exceeded.

    Error: locator.waitFor: Test timeout of 60000ms exceeded.
    Call log:
      - waiting for locator('//span[text()=\'Already exists\']') to be visible


       95 |         });
       96 |         await test.step('Deberia aparecer un mensaje de error', async () => {
    >  97 |             await adminPage.userNameAlreadyExists.waitFor({ state: 'visible' });
          |                                                   ^
       98 |         });
       99 |             
      100 |     });
        at C:\Programacion\QA\Automation\Playwright-ts\Proyecto OrangeHRM\OrangeHRM-Demo\tests\e2e\permissons\add-permissons.spec.ts:97:51
        at C:\Programacion\QA\Automation\Playwright-ts\Proyecto OrangeHRM\OrangeHRM-Demo\tests\e2e\permissons\add-permissons.spec.ts:96:20
]]>
</failure>
</testcase>
</testsuite>
</testsuites>